js/toolbar/z-plane-list.js

```diff
@@ -426,11 +485,28 @@ function ZPlaneList(parent) {
      */
     this._renderZPlaneInfo = function () {
         var zPlaneInfo = document.getElementById('z-plane-info-container');
+
+
+        if (_self.canUpdateDefaultZIndex == true) {
```

Comment:
Improve this part. It's ugly. You don't need `last-button` class. You can select the last child in css [like this](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child). Then you don't need to repeat the `jumpButton`.